contracts/LifMarketMaker.sol

```diff
@@ -97,22 +109,64 @@ contract LifMarketMaker is Ownable {
     ];
 
     uint256 accumDistribution = 0;
-    uint256 periodStartBlock = startBlock;
+    uint256 deltaDistribution = 0;
+    uint256 startBlockPeriod = startBlock;
+
+    require(marketMakerPeriods.length == 0);
 
     for (uint8 i = 0; i < totalPeriods; i++) {
-      uint256 deltaDistribution;
+
       if (totalPeriods == 24) {
         deltaDistribution = deltas24[i];
       } else {
         deltaDistribution = deltas48[i];
       }
+
       accumDistribution = accumDistribution.add(deltaDistribution);
-      uint256 periodEndBlock = periodStartBlock.add(blocksPerPeriod).sub(1);
 
-      distributionPeriods.push(DistributionPeriod(
-        periodStartBlock, periodEndBlock, deltaDistribution, accumDistribution
+      uint256 endBlockPeriod = startBlockPeriod.add(blocksPerPeriod).sub(1);
```

Comment:
as we moved the `deltaDistribution` var declaration out from the for loop, we should also move `endBlockPeriod` out for consistency, no?